### DISCUSSION TITLE
CefSharp.Offscreen to download files from the URL is not working when called from the task scheduler

### DISCUSSION THREAD

**Discussion by sarikachakko** [cefsharp/CefSharp#4715] [Q&A]
I have a windows forms project in vb.net to download csv files from a URL using CefSharp.Offscreen. I have created a task in task scheduler to call this project to download the csv files using CefSharp. But the download is not functioning when it is called from task scheduler and no errors/exceptions were logged. The same code when called from Form_load or button click events is working and file download is happening. I suspect that the code written in CustomDownloadHandler class which implemented the IDownloadHandler interface is not getting executed when called from the task scheduler. Could you please provide a solution in vb.net to make the csv files download from the URL using cefsharp.offscreen

**Comment by ghost** [comment]
Which account is configured to run the scheduled task? LOCAL_SERVICE would be wrong, NETWORK_SERVICE would be right, a regular user account seems right as well. Is there any information in the History tab of the scheduled task?

**Reply by sarikachakko** [reply]
In the history tab of scheduled task, it is showing like: Task Scheduler successfully completed task with return code 0.
